scripts/run_frontend_tests.sh

```diff
@@ -41,6 +41,19 @@ source $(dirname $0)/setup_gae.sh || exit 1
 # TODO(sll): Make this work with fewer third-party dependencies.
 bash scripts/install_third_party.sh
 
+#build so as to have minified js and css
```

Comment:
Maybe:

    # Ensure that generated JS and CSS files are in place before running the tests.